packages/cli/src/commands/bundle/__tests__/getAssetDestPathAndroid-test.ts

```diff
@@ -61,25 +66,44 @@ describe('getAssetDestPathAndroid', () => {
 
   it('should put non-drawable resources to `raw/`', () => {
     const asset = {
+      hash: '565eb945bdbabf6715f01965374ceea3',
       name: 'video',
       type: 'mp4',
       httpServerLocation: '/assets/app/test',
     };
 
     expect(getAssetDestPathAndroid(asset, 1)).toBe(
-      path.normalize('raw/app_test_video.mp4'),
+      path.normalize('raw/app_test_565eb945bdbabf6715f01965374ceea3.mp4'),
```

Comment:
The change of behavior for regular files is not what we expect here I think. Can we fall back to using a hash only if `name` contains illegal characters?

Reply:
@thymikee this is a good idea.